src/pages/community-page.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { Link, graphql } from 'gatsby';
+// import dompurify from 'dompurify';
+import { CommunityPage } from '../types';
+import { ReleaseData } from '../hooks/useReleaseHistory';
+import Layout from '../components/Layout';
+import '../styles/community-page.scss';
+
+interface Props {
+  release?: ReleaseData;
+}
+
+interface CommunityProps {
+  data: CommunityPage;
+}
+export default function DownloadPage({ data }: CommunityProps): JSX.Element {
+  const title = 'Community Page';
+  const description = 'Community page';
+  const commCommData = data.page.html;
+  // Commented out temporarly
+  // const sanitizer = dompurify.sanitize;
+  return (
+    <Layout title={title} description={description}>
+      <main>
+        <div className="container">
+          <div className="community-header community">
+            <CommmunityHeader />
+          </div>
+          <div className="sidebar">
+            <ul>
+              <li className="community-links">
+                <Link to="/">Link 1</Link>
```

Comment:
I believe these should be actual links, otherwise, the production site would look like a work in progress.